runtime/runtime-params-estimator/test-contract/src/lib.rs

```diff
@@ -0,0 +1,571 @@
+#![no_std]
```

Comment:
i guess this file can be copy as part of build.sh, to avoid maintain server copy of it

Reply:
I am not sure I understand what you mean. Could you rephrase it?